vuepress/docs/next/tutorials/devops/change-default-datasource.md

```diff
@@ -1,74 +1,53 @@
-# Change Default Datasources and Connections
-
-## Purpose
-
-This tutorial explains how to customize our EAP- and Wildfly-based Docker images, and then configure the EntandoApp custom resource to use these customizations.
-
-## Requirements
+---
+sidebarDepth: 2
+---
 
--   The Entando Operator is running in the target names
-
--   Keycloak and the Entando Cluster Infrastructure have been deployed
-
--   The required external databases and other services are up and running
+# Change Default Datasources and Connections
 
--   Your own custom project as described in [this tutorial](./build-core-image.md) 
+This tutorial explains how to customize the Tomcat-based Docker images to modify the datasources, and then configure the EntandoApp custom resource to use these customizations.
 
-## Steps
+## Prerequisites
 
-### 1. Create a Docker project for your own base image to Entando's standard base image(s)
-Create a project, ideally in a Git repository, and put the following files in the folder.
-#### 1.1. The standalone.xml/standalone-openshift.xml file of choice 
-It is essential for your custom Docker image to use a file from one of the official Entando Docker base images.
+- A running instance of Entando with the required external databases. [Install Entando on any Kubernetes provider](../#operations) or see [Getting Started](../../docs/getting-started/) for more information. 
 
-If you intend to provide your own EAP-based image, please use this 
-[standalone-openshift.xml](https://github.com/entando/entando-app-engine-base/blob/develop/eap/standalone-openshift.xml)
-file. 
+- Your own custom project as described in [this tutorial](./build-core-image.md) 
 
-If you intend to provide your own Wildfly-based image, please use this 
-[standalone.xml](https://github.com/entando/entando-app-engine-base/blob/develop/wildfly/contrib/wildfly-configuration/standalone.xml)
-file.
+## Customize the Docker Image
+Create a Docker project for your base image using Entando's standard base image. Add the following files to the project folder, typically to a repository like GitHub.  
 
-Please note that if you create your own base image with a different version of EAP or Wildfly, using these files 
-may have unintended consequences.
+**Is server.xml the correct one?**
+1. Using the following Entando template for the Docker image, create the `server.xml`. 
+ 
+If you intend to provide your own Tomcat-based image, please use this 
+[server.xml](https://github.com/entando/entando-app-engine-base/blob/develop/tomcat/conf/server.xml)  
```

Comment:
@pietrangelo should they use this as a base template?

Reply:
See above